File: forge/setup-mod-hook/src/main/java/settingdust/modsets/forge/platform/PlatformHelperForge.java
package settingdust.modsets.forge.platform;

import net.minecraftforge.fml.loading.FMLPaths;
import org.jetbrains.annotations.NotNull;
import settingdust.modsets.PlatformHelper;

import java.nio.file.Path;

public class PlatformHelperForge implements PlatformHelper {

    @NotNull
    @Override
    public Path getConfigDir() {
        return FMLPaths.CONFIGDIR.get().resolve("modsets");
    }
}

File: forge/src/main/kotlin/settingdust/modsets/forge/PlatformHelperForge.kt
package settingdust.modsets.forge

import net.minecraftforge.fml.loading.FMLPaths
import settingdust.modsets.PlatformHelper
import java.nio.file.Path

class PlatformHelperForge : PlatformHelper {
    override val configDir: Path
        get() = FMLPaths.CONFIGDIR.get()
}
